user/enterprise/operations-manual.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Travis CI Enterprise Operations manual
+layout: en_enterprise
+
+---
+
+This document is meant as a guideline to follow in case your Travis CI Enterprise production installation becomes unresponsive or unavailable.
+
+If this happens, we'd like to ask you to follow the steps outlined in this document first before you get in touch with us to save important data and collect as much evidence as possible. This will help us to provide you the best support possible and focus completely on resolving the problem.
+
+## First: Don't panic
+
+The first thing you should do is: keep calm. We're here to help and we're confident that we can bring it back online together with you!
+
+## Prerequisites
+
+Please make sure that you have root access on all machines Travis CI Enterprise is running on. This is crucial because we will ask you to perform operations where you either need to be root or you need to have the appropriate `sudo` privileges. If you don't have direct root privileges, please make sure that a person who has is available.
+
+## Coordinating with Support
+
+We are here to help troubleshoot and fix any problems that may arise with your Travis CI Enterprise installation. For efficiency, though, we would request you minimize making changes to your setup that are not directly recommended by support, or document these steps and notify support accordingly.
```

Comment:
This is seems not really relevant given the specified purpose of the document

Reply:
I know what @schultyy's going for, and I think the concept (please coordinate with support if there's an issue, and if you are, document changes as you go) is pretty important... but, the fact it's not obvious is definitely an issue... 

Another idea: What if all of the stuff about support appeared with the support bundle discussion? That's really the only place that it matters - if the customer's gonna get in contact, we're gonna (most likely) need a support bundle, so the steps on how to coordinate with support are moot if that's not part of the troubleshooting

Reply:
Are we talking about changes made trying to troubleshoot, or just "changes to your installation"? Either way, moving into the support bundle makes sense,